src/time_tracker/invoices/spec.clj

```diff
@@ -0,0 +1,112 @@
+(ns time-tracker.invoices.spec
+  (:require [clojure.spec :as s]
+            [clojure.spec.gen :as gen]
+            [time-tracker.users.spec :as users-spec]
+            [time-tracker.spec :as core-spec]
+            [time-tracker.util :as util]))
+
+(s/def ::hours (core-spec/positive-bigdec 2))
+
+(s/def ::rate ::core-spec/money-val)
+(s/def ::user-id ::users-spec/id)
+(s/def ::user-id->rate-map (s/keys :req-un [::user-id ::rate]))
```

Comment:
This could be `user-rate-map`, similar to https://github.com/nilenso/time-tracker/pull/63/files#diff-943884db8e1671a3fb4761732d5687dfR51